test/e2e/utils/common.go

```diff
@@ -494,3 +496,66 @@ func userAndPassword(i *ispnv1.Infinispan, kube *TestKubernetes) (*string, *stri
 	ExpectNoError(err)
 	return &user, &pass
 }
+
+func CreateCache(cacheName string, encoding mime.MimeType, client HTTPClient) {
+	config := fmt.Sprintf("{\"distributed-cache\":{\"mode\":\"SYNC\",\"remote-timeout\": 60000,\"encoding\":{\"media-type\":\"%s\"}}}", encoding)
+	NewCacheHelper(cacheName, client).Create(config, mime.ApplicationJson)
+}
+
+// addData Populates a cache with bounded parallelism
+func AddDataToPlainCache(cacheName string, entries int, client HTTPClient) {
```

Comment:
The comment needs to use the same format as the function name, i.e. `AddDataToPlainCache populates ...`